plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+""" Amazon Cognito Identity implementation without external dependencies """
```

Comment:
What is this used for?

Reply:
Amazon Cognito is the authentication service used by GoPlay. You authenticate with amazon cognito, and you get a token back that you can use to talk to the API.

It's also used by Retrospect of @basrieter

Reply:
That is correct. They use AWS as identity provider.